lib/message.rb

```diff
@@ -0,0 +1,25 @@
+require 'twilio-ruby'
+
+class Message
+
+  def complete_order(total_price)
+    send_text("Thank you for your order: £#{total_price}. Expect your food by #{delivery_time} ")
+  end
+
+  def send_text(message)
+  	account_sid = 'ACa4cf08b22ebab768c156ad454eaccb71' 
+	  auth_token = '3321406ad1dfa603ec0195d363a5fbef' 
+		@client = Twilio::REST::Client.new account_sid, auth_token 

```

Comment:
use dependency injection to inject the client to reduce dependency and allow easier testing